app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belong_to :posts
-  belong_to :users
+  belong_to :post
+  belong_to :user
```

Comment:
- [ ] [REQUIRED] Let's be consistent with our naming conventions. As you have used in the `Post` table `author` with a class name of `User`. Let's implement the same here such that your code looks like this:
`belongs_to :author, class_name: 'User'`
The association between models are really important.